src/Cake.GitHub.Tests/Helpers/GitHubClientMock.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Moq;
+using Octokit;
+
+namespace Cake.GitHub.Tests
+{
+    internal class GitHubClientMock
+    {
+        public class RepositoriesClientMock
+        {
+            private readonly Mock<IRepositoriesClient> m_Mock = new Mock<IRepositoriesClient>(MockBehavior.Strict);
```

Comment:
I think we don't use the `m_` prefix anywhere else?

Reply:
That convention is used in the code bases I usually work on, so I just used it without thinking about it, I've updated all fields to use just a `_` prefix